notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-095_settling_big_vibrations.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<p>Should this cell be deleted ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
We eliminated this version of the notebook.